droid-hal-device-img-boot.inc

```diff
@@ -11,6 +11,9 @@
 #                %{devicetree}" to build with the standard binaries from
 #                droid-hal-device kernel package.
 #
+# root_part_label:	Sailfish OS parition label.
+# factory_part_label:	Factory image partition label.
```

Comment:
Actually, only one string is allowed, because we use these also on initrd-helpers scripts.